src/include/autoinstall/autopart.rb

```diff
@@ -271,6 +271,12 @@ def preprocess_partition_config(xmlflex)
 
     def try_add_boot(conf, disk)
       conf = deep_copy(conf)
```

Comment:
btw deep copy is not needed here, as eval do copy and conf is not used anywhere else.